client/my-sites/marketing/ultimate-traffic-guide/index.js

```diff
@@ -0,0 +1,288 @@
+import { isFetchingUserPurchases, getUserPurchases } from 'calypso/state/purchases/selectors';
+import { getCurrentUserId } from 'calypso/state/current-user/selectors';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import Experiment, {
+	DefaultVariation,
+	LoadingVariations,
+	Variation,
+} from 'calypso/components/experiment';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const WPCOM_TRAFFIC_GUIDE = 'traffic-guide';
+
+export const hasTrafficGuidePurchase = ( purchases ) =>
+	!! (
+		purchases && purchases.find( ( purchase ) => WPCOM_TRAFFIC_GUIDE === purchase.productSlug )
+	);
+
+export const downloadTrafficGuide = () => {
+	window.location.href = 'https://public-api.wordpress.com/wpcom/v2/traffic-guide-download';
+};
+
+export default function UltimateTrafficGuide() {
+	const translate = useTranslate();
+	const userId = useSelector( ( state ) => getCurrentUserId( state ) );
+	const isLoading = useSelector( ( state ) => isFetchingUserPurchases( state ) );
+	const purchases = useSelector( ( state ) => getUserPurchases( state, userId ) );
+	const hasPurchase = hasTrafficGuidePurchase( purchases );
+
+	const renderContent = () => {
+		return (
+			<>
+				<PageViewTracker
+					path={ '/marketing/ultimate-traffic-guide' }
+					title={ 'Ultimate Traffic Guide' }
+					properties={ { has_purchase: hasPurchase } }
+				/>
+				{ hasPurchase ? (
+					<DownloadPage translate={ translate } />
+				) : (
+					<SalesPage translate={ translate } />
+				) }
+			</>
+		);
+	};
+
+	return (
+		<CompactCard className="ultimate-traffic-guide">
+			{ ! purchases && <QueryUserPurchases userId={ userId } /> }
+			{ isLoading || ! purchases ? <Placeholder num={ 5 } /> : renderContent() }
+		</CompactCard>
+	);
+}
+
+const Placeholder = ( { num = 1 } ) =>
+	[ ...Array( num ) ].map( ( e, i ) => (
+		<div key={ `${ i }` } className="ultimate-traffic-guide__placeholder">
+			<div className="ultimate-traffic-guide__placeholder-animation"></div>
+		</div>
+	) );
+
+const SalesPage = ( { translate } ) => {
+	const cost = useSelector( ( state ) => getProductDisplayCost( state, WPCOM_TRAFFIC_GUIDE ) );
+	const isLoading = useSelector( ( state ) => isProductsListFetching( state ) );
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
+
+	const defaultVariation = () => (
+		<>
+			<h1 className="ultimate-traffic-guide__header">
+				{ translate( 'Introducing the WordPress.com Ultimate Traffic Guide' ) }
+			</h1>
+			<h2 className="ultimate-traffic-guide__sub-header">
+				{ translate( 'Discover today’s most important traffic secrets' ) }
+			</h2>
+			<p>
+				{ translate(
+					'We developed this 96 page guide to teach you every modern website traffic trick you need to know in 2020 and beyond.'
```

Comment:
It will soon become 2021. Do you think we should make the year dynamic?

Reply:
Good catch! Made this dynamic.